docker-compose-dev.yml

```diff
@@ -164,24 +163,24 @@ services:
     tmpfs:
       - /mail:mode=770,size=128M,uid=1000,gid=1000
 
-  dev_keycloak:
-    container_name: "dev_keycloak"
-    image: 'quay.io/keycloak/keycloak:21.0'
+  keycloak:
```

Comment:
Should we stick with "dev_keycloak" since this is the "dev" compose file?

We do have another compose file over at docker/compose/demo/compose.yml if we want to add a non-dev container to it. It's used in this tutorial: https://guides.dataverse.org/en/6.4/container/running/demo.html